README.md

```diff
@@ -22,4 +22,4 @@ Learn the implemented App and the example and reimplement it with Redux having t
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - This task does not have tests yet!
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_redux-list-of-posts/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://GGLUTT.github.io/react_redux-list-of-posts/) and add it to the PR description.
```

Comment:
Add this demo link to the PR description 
<img width="804" alt="image" src="https://github.com/user-attachments/assets/cba9bee1-f0d5-4462-a7e8-56a5caa18acf">